src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,41 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => {
+          const isSelected = () => {
+            return tab.id === selectedTab.id;
+          };
```

Comment:
```suggestion
          const isSelectedTab = tab.id === selectedTab.id
```

can be more simplify